lib/auth.js

```diff
@@ -118,8 +118,10 @@ module.exports = {
       app_token: appToken
     };
 
-    this._authenticate(requestData, function(responseData) {
-      this._onAccessTokenAcquired(responseData, callback);
+    this._authenticate(requestData, function(err, responseData) {
+      if (err === null) {
+        this._onAccessTokenAcquired(responseData, callback);
+      }
```

Comment:
We need to let pass the `err` all the way to the top, like this: https://github.com/podio/podio-js/blob/master/lib/auth.js#L104